src/g3log/time.hpp

```diff
@@ -60,7 +60,10 @@ namespace g3 {
    inline system_time_point to_system_time(const high_resolution_time_point& ts)
    {
 	   using namespace std::chrono;
-	   return time_point_cast<system_clock::duration>(system_clock::now() + (ts - high_resolution_clock::now()));
+	   static auto hrs_now = high_resolution_clock::now();
```

Comment:
Can you make these function local static members also `const` to further explain there `once and won't change` behavior.